src/main.rs

```diff
@@ -4,11 +4,13 @@
 #![feature(bind_by_move_pattern_guards)]
 #![feature(const_str_as_bytes)]
 #![feature(box_syntax)]
+#![feature(repeat_generic_slice)]
```

Comment:
where do we need this? Wouldn't a `iter::repeat(X).take(N).collect()` (maybe with flatten) work?

Reply:
in the asciifile tests. You used repeat for str, and I changed everything to byte strings `b""` => needed repeat thre